sa/saro.go

```diff
@@ -194,7 +194,7 @@ func (ssa *SQLStorageAuthorityRO) CountRegistrationsByIP(ctx context.Context, re
 			"latest":   time.Unix(0, req.Range.Latest),
 		})
 	if err != nil {
-		return &sapb.Count{Count: -1}, err
+		return &sapb.Count{Count: 0}, err
```

Comment:
We're returning a reference to an object here -- why not return nil, like we do on line 180?

Reply:
Oh, fair!